requirements.txt

```diff
@@ -1 +1,2 @@
-v8-cffi==0.2.0
+six==1.10.0
+-e git+https://github.com/heston/v8-cffi.git@e1693986f7ea5e96680bac3e46afb2352b23a133#egg=v8_cffi
```

Comment:
TODO: Will updated this when https://github.com/nitely/v8-cffi/pull/12 is merged.